sysvinit/openrc/cloud-init

```diff
@@ -1,4 +1,7 @@
 #!/sbin/openrc-run
+
+description="cloud-init main stage"
```

Comment:
I think that "cloud-init network stage" would be better in the long run.

Naming is hard, and unfortunately not something that cloud-init has historically excelled at. Naming one of the stages the same thing as the project name was a blunder, in my opinion, but probably easier seen in hindsight. In the docs we call this the "network stage", and I think that this is probably the safest name to standardize showing to users. The "init stage" is confusingly a substring of the project name, the "cloud-init stage" is confusingly _the project name_, and "main stage" is also confusing (imo) - what makes that the main stage?

Sidenote: I wrote up a [proposal a while back](https://discourse.ubuntu.com/t/proposed-specification-cloud-init-service-name-changes/40125) that would make cloud-init's stage naming system much more coherent. I haven't actually started a conversation about whether it is worth pursuing with the community since I'm really not sure that it would be worth it, but it describes some of the issues I see surrounding stage naming.

Reply:
Ok I've changed this.